coalib/parsing/CliParsing.py

```diff
@@ -87,3 +87,31 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(cli_sections, log_printer):
```

Comment:
Can we make this a little generic to check for conflicts in any section ?
And possible pass `conflicts` into it ?

We should have a fucntion in DefaultArgParser like `cli_conflicts()` which gives `conflicts`, so that the logic for arg parsing is in the same place ?